code/modules/clothing/accessories/badges.dm

```diff
@@ -6,7 +6,7 @@
 
 /obj/item/clothing/accessory/badge
 	name = "Inspector's badge"
```

Comment:
Should give this a different name since "Inspector's badge" is already used.

Reply:
Better idea, what if I just made the old inspector's badge distinctly old?

Reply:
Now they're distinct, boom